bower.json

```diff
@@ -17,6 +17,11 @@
   ],
   "dependencies": {
     "jquery": "3.2.1",
-    "bootstrap": "3.3.7"
+    "bootstrap": "3.3.7",
+    "nvd3": "^1.8.5",
```

Comment:
Curious as to why you added these? If you added them in anticipation of experimenting with d3, I think a better way to do that would be to create a separate branch.